demo/README.md

```diff
@@ -9,125 +9,13 @@ To run the completed project in this folder, you need the following:
 
 If you don't have a Microsoft account, you can [sign up for the Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program) to get a free Microsoft 365 subscription.
 
-## Register an application
+## Configure the sample for user authentication
```

Comment:
Would it be prudent to specify "delegated"?

```suggestion
## Configure the sample for [user delegated authentication](https://docs.microsoft.com/en-us/graph/auth/auth-concepts#delegated-and-application-permissions)
```

Reply:
Azure docs for authentication/authorization don't mention "delegated". Delegated is a permission type. I will a link to the doc in this section though.